tests/meta/workflow/mutant/test_mutant_quality_controller.py

```diff
@@ -0,0 +1,143 @@
+from pathlib import Path
+from cg.meta.workflow.mutant.quality_controller.models import (
+    QualityMetrics,
+    QualityResult,
+    SampleQualityResult,
+)
+from cg.meta.workflow.mutant.quality_controller.quality_controller import QualityController
+from cg.store.models import Case, Sample
+
+
+def test_quality_control_case_qc_pass(
+    case_qc_pass: Case,
+    quality_controller: QualityController,
+    mutant_analysis_dir_case_qc_pass: Path,
+    mutant_results_file_path_qc_pass: Path,
+):
+    # GIVEN a case object and a corresponding case_results_file_path
+
+    # WHEN performing qc
+
+    case_qc: QualityResult = quality_controller.quality_control(
+        case=case_qc_pass,
+        case_path=mutant_analysis_dir_case_qc_pass,
+        case_results_file_path=mutant_results_file_path_qc_pass,
+    )
+
+    # THEN no error is thrown
+    assert case_qc.case.internal_negative_control_passes_qc is True
+    assert case_qc.case.external_negative_control_passes_qc is True
+    assert case_qc.passes_qc is True
+
+
+def test_quality_control_case_qc_fail(
+    case_qc_fail: Case,
+    quality_controller: QualityController,
+    mutant_analysis_dir_case_qc_fail: Path,
+    mutant_results_file_path_qc_fail: Path,
+):
+    # GIVEN a case object and a corresponding case_results_file_path
+
+    # WHEN performing qc
+
+    quality_controller.quality_control(
+        case=case_qc_fail,
+        case_path=mutant_analysis_dir_case_qc_fail,
+        case_results_file_path=mutant_results_file_path_qc_fail,
+    )
+
+    # THEN no error is thrown
```

Comment:
Would we not want to check that the passes_qc statuses are different than in the test before?

Reply:
Or I am a bit unsure of what behaviour we are testing here.